packages/deno/test/mod.test.ts

```diff
@@ -0,0 +1,76 @@
+import { assertEquals } from 'https://deno.land/std@0.202.0/assert/assert_equals.ts';
+import { assertSnapshot } from 'https://deno.land/std@0.202.0/testing/snapshot.ts';
+import type { Event, Integration } from 'npm:@sentry/types';
+import { createStackParser, nodeStackLineParser } from 'npm:@sentry/utils';
+
+import { defaultIntegrations, DenoClient, Hub, Scope } from '../build/index.js';
+import { getNormalizedEvent } from './normalize.ts';
+import { makeTestTransport } from './transport.ts';
```

Comment:
m: We can configure the tsconfig to adjust the resolution of these imports to allow for `.ts` endings. Right now TS is screaming.

Reply:
I'll have a look. I have the Deno vscode extension installed so don't see these errors!

Reply:
`allowImportingTsExtensions` is TypeScript 5+ and we're on 4.9.5.

For now I've added the Deno vscode extension to the recommended extensions and only enabled if for `packages/deno/test`.

I've also had to disable eslint for the `./scripts` directory because the parser can't deal with top level await.

Reply:
Okok this also makes sense. I briefly discussed in the team whether we should bump the repo TS version to 5 but there are some considerations with regards to how we build types. Let's stick with the extension for now! Thanks for experimenting here.